src/System.Runtime/tests/Performance/Perf.Double.cs

```diff
@@ -13,16 +11,30 @@ namespace System.Tests
 {
     public class Perf_Double
     {
+        // NOTE: Consider duplicating any tests added here in Perf.Single.cs
+
         private volatile string _string;
 
         [Benchmark]
-        [InlineData(104234.343, 1_000_000)]
-        [InlineData(double.MaxValue, 100_000)]
-        [InlineData(double.MinValue, 100_000)]
-        [InlineData(double.MinValue / 2, 100_000)]
-        [InlineData(double.NaN, 10_000_000)]
-        [InlineData(double.PositiveInfinity, 10_000_000)]
-        [InlineData(2.2250738585072009E-308, 100_000)]
+        [InlineData(double.NegativeInfinity, 10_000_000)]   // Negative Infinity
```

Comment:
What's the method for deciding the innerIterations value here? It seems almost random.

Reply:
I followed the existing numbers that were used. They seemed to be setup for:
`10m` is used for values that are covered by a "fast path"
`1m` is used for values in a "standard input range"
`100k` is used for values that are "edge cases" (and generally require longer processing time)